src/hatch_fancy_pypi_readme/_config.py

```diff
@@ -19,36 +22,97 @@ class Config:
     substitutions: list[Substituter]
 
 
+SCHEMA = {
+    "type": "object",
```

Comment:
It's somewhat decent practice to put
```suggestion
    "$schema": DRAFT202012Validator.META_SCHEMA["$id"],
    "type": "object",
```

in here (i.e. for the schema to self-descriptively say what version it uses).

Reply:
there's a typo i won't commit it directly but thanks :)